### ISSUE TITLE
3.0.0-alpha.3 index.cjs doesn't export __debug API

### ISSUE THREAD

**Issue by Janther** [prettier/prettier#13730]
Hi I'm working on the integration and to run our tests we were using it to compare AST.

it was available in `3.0.0-alpha.0`

**Environments:**

- Prettier Version: 3.0.0-alpha.3
- Running Prettier via: API 
- Runtime: Node.js v14 
- Operating System: macOS  
- Prettier plugins (if any): prettier-plugin-solidity 

**Steps to reproduce:**

```Bash
git clone git@github.com:prettier-solidity/prettier-plugin-solidity.git
cd prettier-plugin-solidity
git checkout preparing-parser-for-v3
npm install
npm run test:all
```

**Expected behavior:**

Test should pass

**Actual behavior:**

error `Cannot read property 'parse' of undefined`

**Comment by fisker** [comment]
Why don't you try import the esm version?

**Comment by fisker** [comment]
Fixed in [v3.0.0-alpha.4](https://github.com/prettier/prettier/releases/tag/3.0.0-alpha.4)
